### ISSUE TITLE
When we convert resistance to temperature we cannot read below 35 deg

### ISSUE THREAD

**Issue by shortyishere** [SequentMicrosystems/megabas-rpi#8]
The resistance stops at 30,000 ohms?

**Comment by alexburcea2877** [comment]
You have right, we limited the resistance value to 30K because of the high errors in measurements above that value.

**Comment by shortyishere** [comment]
Agreed. Inaccurate numbers would give us something to see if it goes below freezing on a typical type 2 10k thermistor which is used by most manufactures. Is there a way we can adjust this if needed. And maybe a firmware that we could use with this enabled.

**Comment by alexburcea2877** [comment]
No problem to increase that limit in firmware if you need it. I will let you know when is ready

**Comment by alexburcea2877** [comment]
We released a new firmware version, the resistance limit has been increased until 60K ohms. Please update the card firmware, instructions here: https://github.com/SequentMicrosystems/megabas-rpi/blob/master/update/README.md

**Comment by shortyishere** [comment]
Awesome that should be good for our needs. Appreciate it!

**Comment by alexburcea2877** [comment]
I think we overreacted  by setting the limit to 30K.  Thank you for pointing this.
